### PR TITLE
Align flow with design

### LINKED ISSUE #104: align the registration flow with designs

### PR DESCRIPTION
**Pull request by cezaryskura**
[close #104]
- remove unnecessary fields from business Owner registration flow
- add address check-in business owner flow
- add fields to confirm password and email on sign up page
- small adjustments on styles

### CODE REVIEW

File: src/components/entrepreneur-flow/EntrepreneurFlowBusinessInfo.vue [hunk 1]
```diff
@@ -23,27 +15,36 @@
       field-label="Huis nummer"
       :required="true"
     />
+    <div v-if="isAddressChecked" class="pt-5">
```
Comment (IAfanasovMob) [inline src/components/entrepreneur-flow/EntrepreneurFlowBusinessInfo.vue:18]: what if we can not resolve the address? do we have any message to help a user recover from the situation?

File: src/components/entrepreneur-flow/EntrepreneurFlowBusinessInfo.vue [hunk 2]
```diff
@@ -70,24 +74,45 @@ export default {
...
     },
+    async checkAddress() {
+      const zipCode = this.entrepreneur.address.zipCode
+      const houseNumber = this.entrepreneur.address.houseNumber
+
+      if (!zipCode || !houseNumber) {
+        return
+      }
+
+      this.isAddressChecked = true
```
Comment (IAfanasovMob) [inline src/components/entrepreneur-flow/EntrepreneurFlowBusinessInfo.vue:92]: does this flag mean `is address confirmed by used`?

File: src/components/entrepreneur-flow/EntrepreneurFlowBusinessInfo.vue (hunk 1 above)
Comment (cezaryskura) [reply src/components/entrepreneur-flow/EntrepreneurFlowBusinessInfo.vue:18]: shall we proceed if we can't resolve address?

File: src/components/entrepreneur-flow/EntrepreneurFlowBusinessInfo.vue (hunk 2 above)
Comment (cezaryskura) [reply src/components/entrepreneur-flow/EntrepreneurFlowBusinessInfo.vue:92]: No, this flag is for address check.
User will confirm this address by clicking 'Volgende'

File: src/components/entrepreneur-flow/EntrepreneurFlowBusinessInfo.vue (hunk 1 above)
Comment (IAfanasovMob) [reply src/components/entrepreneur-flow/EntrepreneurFlowBusinessInfo.vue:18]: I would say `no` for the MVP at least
the company would not be shown on the map and there will be no way for user to find the entrepreneur except for using the direct link

File: src/components/entrepreneur-flow/EntrepreneurFlowBusinessInfo.vue (hunk 2 above)
Comment (IAfanasovMob) [reply src/components/entrepreneur-flow/EntrepreneurFlowBusinessInfo.vue:92]: hard to come up with a clear variable name.... checked confuses between "checked by the backend" and "checked by the user". 
"resolved" would be also confusing - even if BE does not resolve the address, this variable still will be true... I give up 🙃

**Comment by IAfanasovMob** [comment]
Feels like we need to have a flow to deal with an incorrect address entered by the user

**Pull request merged**